CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 🚩 Commands and flags
 
+- Adds new command: `calibre site create-pull-request-review` to compare a preview deployment for a Site already monitored by Calibre.
+- Adds new command: `calibre site pull-request-reviews` to list Pull Request reviews.
+- Adds new command: `calibre site pull-request-review` to retrieve / display an existing Pull Request review.
```

Comment:
```suggestion
- Adds new command: `calibre site pull-request-review` to retrieve and display an existing Pull Request Review.
```

Reply:
you might want to consider linking Pull Request Review to docs at least once since it's mentioned quite a bit here